react/game_area.js

```diff
@@ -185,7 +187,13 @@ class GameArea extends Component {
       });
       
       if (unfinishedEnvelopes.length > 0) {
-        this.createStartEnvelope(unfinishedEnvelopes);
+        if (unfinishedEnvelopes[0].prevCompleted && !unfinishedEnvelopes[0].isChanged) {
+          console.log("Inside the if statement");
```

Comment:
Probably take out the debugging here

Reply:
Just updated--take a look when you get a chance.